homeassistant/helpers/state.py

```diff
@@ -10,6 +10,7 @@
 from typing import Any
 
 from homeassistant.components.sun import STATE_ABOVE_HORIZON, STATE_BELOW_HORIZON
+from homeassistant.components.twitch.sensor import STATE_OFFLINE, STATE_STREAMING
```

Comment:
This should not be solved in this helper, instead, contain/solve it within the twitch integration itself.

Reply:
Thanks for checking, I'm going to close this pull request.